includes/api/class-edd-api-v2.php

```diff
@@ -317,4 +317,111 @@ public function get_customers( $args = array() ) {
 		return $customers;
 	}
 
+	/**
+	 * Retrieves Recent Sales
+	 *
+	 * @access public
+	 * @since  2.6
+	 * @return array
+	 */
+	public function get_recent_sales() {
+		global $wp_query;
+
+		$sales = array();
+
+		if( ! user_can( $this->user_id, 'view_shop_reports' ) && ! $this->override ) {
+			return $sales;
+		}
+
+		if( isset( $wp_query->query_vars['id'] ) ) {
+			$query   = array();
+			$query[] = new EDD_Payment( $wp_query->query_vars['id'] );
+		} elseif( isset( $wp_query->query_vars['purchasekey'] ) ) {
+			$query   = array();
+			$query[] = edd_get_payment_by( 'key', $wp_query->query_vars['purchasekey'] );
+		} elseif( isset( $wp_query->query_vars['email'] ) ) {
+			$query = edd_get_payments( array( 'fields' => 'ids', 'meta_key' => '_edd_payment_user_email', 'meta_value' => $wp_query->query_vars['email'], 'number' => $this->per_page(), 'page' => $this->get_paged(), 'status' => 'publish' ) );
+		} else {
+			$query = edd_get_payments( array( 'fields' => 'ids', 'number' => $this->per_page(), 'page' => $this->get_paged(), 'status' => 'publish' ) );
+		}
+
+		if ( $query ) {
+			$i = 0;
+			foreach ( $query as $payment ) {
+				if ( is_numeric( $payment ) ) {
+					$payment = new EDD_Payment( $payment );
+				}
+
+				$payment_meta = $payment->get_meta();
+				$user_info    = $payment->user_info;
+
+				$sales['sales'][ $i ]['ID']             = $payment->number;
+				$sales['sales'][ $i ]['transaction_id'] = $payment->transaction_id;
+				$sales['sales'][ $i ]['key']            = $payment->key;
+
+				$discounts       = ! empty( $payment->discounts ) ? explode( ',', $payment->discounts ) : array();
+				$discount_values = array();
+
+				foreach ( $discounts as $discount ) {
+					if ( 'none' === $discount ) {
+						continue;
+					}
+
+					// Clean up the discount code for use in the JSON
+					$code  = trim( $discount );
+					$total = $payment->subtotal;
+
+					if ( edd_use_taxes() && edd_prices_include_tax() ) {
+						$total += $payment->tax;
+					}
+
+					$discount_values[ $code ] = (float) $total - ( edd_get_discounted_amount( $code, $total ) );
```

Comment:
This won't always be correct since some discounts are limited to specific items.

Example, if a discount is limited to item A but an order has item A and B, using the subtotal of the order here will give incorrect amounts.